j/jszip/jszip_ubi8.sh

```diff
@@ -1,9 +1,12 @@
+#!/bin/bash -e
 # ----------------------------------------------------------------------------
 #
 # Package               : jszip
-# Version               : 3.3.0
+# Version               : 3.3.0 , 3.7.1
 # Source repo           : https://github.com/Stuk/jszip
 # Tested on             : UBI 8
+# Language      	: Node
+# Travis-Check  	: True
 # Script License        : Apache License, Version 2 or later
 # Maintainer            : Swati Singhal <swati.singhal@ibm.com>
```

Comment:
can you also add your name to the maintener?

Reply:
Yes will add it